cordformation/src/test/resources/postgres.gradle

```diff
@@ -0,0 +1,120 @@
+ext {
+    postgresql_version     = '42.2.12'
+    postgres_image_version = '12'
```

Comment:
I don't think we actually test against postgres 12, so probably we should default the postgres version to 11 (which we do test against). (I suspect 12 works just fine, but better safe than sorry)

Reply:
Updated Postgres Docker image version to 11